tests/python/unittest/test_operator.py

```diff
@@ -5702,6 +5702,98 @@ def test_softmax():
     check_softmax_grad(default_context())
     check_smoothed_softmax_grad(default_context())
 
+@with_seed()
+def test_variance():
+  def true_var(x, axis=None):
+    if len(x.shape) == 1:
+      return np.var(x, keepdims=True, axis=axis)
+    else:
+      return np.var(x, axis=axis)
+  def true_var_grad(x, ograd, axis=None):
+    if axis is None:
+      return 2 * (x - np.mean(x)) * ograd / np.prod(x.shape)
+    else:
+      denom = x.shape[axis]
+      ograd_shape = tuple(1 if i == axis else k for i, k in enumerate(x.shape))
+      return 2 * (x - np.mean(x, axis=axis, keepdims=True)) * ograd.reshape(ograd_shape) / denom
+
+  for ndim in range(1, 6):
+    # check forward
+    shape = rand_shape_nd(ndim, 5)
+    data = rand_ndarray(shape=shape, stype='default')
+    data_np = data.asnumpy()
+    expected = true_var(data_np)
+    output = mx.nd.variance(data)
```

Comment:
why not include the test for the imperative interface in `test_ndarray.py` file and use `check_symbolic_forward` here?